js/app.js

```diff
@@ -21,4 +21,8 @@ document.addEventListener("DOMContentLoaded", () => {
       toDoList.renderTasks();
     }
   });
+
+  document.getElementById("btn-all").addEventListener("click", () => {
+    toDoList.renderTasks();
+  });
 });
```

Comment:
perhaps you can add a new function inside the toDoList like `setFilter` and set the filter inside the class once click on one of these buttons and inside the render function you can use that stored filter value to render only the needed tasks